etc/behat/services/pages.xml

```diff
@@ -50,6 +51,9 @@
         <service id="sylius.behat.page.checkout_shipping" class="%sylius.behat.page.checkout_shipping.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false" />
         <service id="sylius.behat.page.checkout_thank_you" class="%sylius.behat.page.checkout_thank_you.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false" />
 
-        <service id="sylius.behat.page.external.paypal_checkout_express" class="%sylius.behat.page.external.paypal_checkout_express.class%" parent="sylius.behat.page" scope="scenario" public="false" />
+        <service id="sylius.behat.page.external.paypal_checkout_express" class="%sylius.behat.page.external.paypal_checkout_express.class%" parent="sylius.behat.page" scope="scenario" public="false" >
+            <argument type="service" id="sylius.repository.payment_security_token" container="symfony" />
+        </service>
+        <service id="sylius.behat.page.order_payments" class="%sylius.behat.page.order_payments.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false" />
```

Comment:
Why do we have to define parent class in service definition? It is not typical for sylius services